api/tests/integration/infrastructure/repositories/skill-repository_test.js

```diff
@@ -108,4 +108,38 @@ describe('Integration | Repository | skill-repository', () => {
       expect(skills).to.deep.include.members([activeSkill, archivedSkill]);
     });
   });
+
+  describe('#assessedDuringCampaignParticipation', () => {
+
+    beforeEach(() => {
+      const learningContent = {
+        skills: [
+          { id: 'skill1', status: 'actif' },
+          { id: 'skill2', status: 'archivé' },
+          { id: 'skill3', status: 'périmé' },
+          { id: 'skill4', status: 'actif' },
+        ],
+      };
+      mockLearningContent(learningContent);
+    });
```

Comment:
**nitpick:** Ajouter un saut de ligne entre le beforeEach et le it

Reply:
done